src/benchmark/operators/table_scan_sorted_benchmark.cpp

```diff
@@ -228,15 +233,18 @@ void registerTableScanSortedBenchmarks() {
     for (const auto selectivity : selectivities) {
       for (const auto& data_type : supported_data_types) {
         for (const auto& mode : modes) {
-          for (const auto is_between_scan : {false, true}) {
-            const auto& table_generator = table_types.at(data_type);
-
-            const std::string between_label = is_between_scan ? "Between" : "";
-            benchmark::RegisterBenchmark(("BM_Table" + between_label + "ScanSorted/" + encoding_name + "/" +
-                                          std::to_string(selectivity) + "/" + data_type + "/" + mode)
-                                             .c_str(),
-                                         BM_TableScanSorted, ROWS, selectivity, encoding_type, mode, is_between_scan,
-                                         table_generator);
+          for (const auto& is_reference_scan : {false, true}) {
+            for (const auto is_between_scan : {false, true}) {
```

Comment:
is there a special reason why we don't iterate over sectivities and is_between_scan by reference?

Reply:
Thanks for the hint, but it's actually wrong to use a reference here and clang-tidy would probably complain as the boolean  `is_trivially_copyable`.